build.gradle

```diff
@@ -198,6 +198,11 @@ dependencies {
   implementation group: 'uk.gov.hmcts.reform', name: 'logging-appinsights', version: versions.reformLogging
   implementation group: 'org.springframework.cloud', name: 'spring-cloud-starter-netflix-hystrix', version: '2.2.2.RELEASE'
 
+  implementation group: 'com.microsoft.azure', name: 'azure-servicebus', version: '1.2.18', {
```

Comment:
any reason not to use `3.1.5`. we do need to migrate old projects at some point?

Reply:
processor using the old one. got the latest one.